Source/ASTextNode2.mm

```diff
@@ -1061,10 +1062,15 @@ - (NSUInteger)lineCount
 - (void)_invalidateTruncationText
 {
   ASLockScopeSelf();
-  _textContainer.truncationToken = nil;
+  [self _locked_invalidateTruncationText];
   [self setNeedsDisplay];
```

Comment:
Is it actually ok to call `-setNeedsDisplay` here?

Reply:
I asked that myself too, but I didn't want to change it within this diff. In theory the code in `ASTextNode2` should be adjusted to the same as in `ASTextNode` after this lands.

Reply:
👍